api/iceberg-service/build.gradle.kts

```diff
@@ -111,3 +112,5 @@ listOf("sourcesJar", "compileJava").forEach { task ->
 sourceSets {
   main { java { srcDir(project.layout.buildDirectory.dir("generated/src/main/java")) } }
 }
+
+tasks.named("javadoc") { dependsOn("jandex") }
```

Comment:
Without this you get:

```
A problem was found with the configuration of task ':polaris-api-iceberg-service:javadoc' (type 'Javadoc').
  - Gradle detected a problem with the following location: '/Users/adutra/Projects/oss/polaris/polaris1/api/iceberg-service/build/resources/main'.
    
    Reason: Task ':polaris-api-iceberg-service:javadoc' uses this output of task ':polaris-api-iceberg-service:jandex' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed.
    
    Possible solutions:
      1. Declare task ':polaris-api-iceberg-service:jandex' as an input of ':polaris-api-iceberg-service:javadoc'.
      2. Declare an explicit dependency on ':polaris-api-iceberg-service:jandex' from ':polaris-api-iceberg-service:javadoc' using Task#dependsOn.
      3. Declare an explicit dependency on ':polaris-api-iceberg-service:jandex' from ':polaris-api-iceberg-service:javadoc' using Task#mustRunAfter.
```

Reply:
Interesting... :thinking: